src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -163,6 +169,13 @@ public boolean isAnyFieldEdited() {
             return CollectionUtil.isAnyNonNull(name, phone, email, role, affiliations);
         }
 
+        /**
+         * Returns true if affiliations is edited.
+         */
+        public boolean isAffiliationEdited() {
+            return CollectionUtil.isAnyNonNull(affiliations);
```

Comment:
Maybe you can simply check using .isEmpty()? Might be more intuitive that way since we are only checking for affiliations.

`return !affiliations.isEmpty();`

Reply:
### Yes, that is better since I am only checking for 1 parameter. Will change in the next commit thanks!